index.html

```diff
@@ -1,14 +1,99 @@
+</g>
+</g>
+<defs>
+<clipPath id="clip0_39_177">
+<rect width="117" height="24" fill="white"/>
+</clipPath>
+<clipPath id="clip1_39_177">
+<rect width="117" height="24" fill="white"/>
+</clipPath>
+</defs>
+</svg>
+
+</div>
+
+<div class="social-media">
```

Comment:
- [ ] Kindly make sure the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.